src/main/java/org/example/springintro/model/Book.java

```diff
@@ -1,29 +1,30 @@
-package org.example.spring_intro.model;
+package org.example.springintro.model;
 
+import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Data
+@SQLDelete(sql = "UPDATE books SET is_delete = true WHERE id=?")
+@Where(clause = "is_delete=false")
+@Table(name = "books")
 public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-//    id (Long, PK)
-//title (String, not null)
-//author (String, not null)
-//isbn (String, not null, unique)
-//price (BigDecimal, not null)
-//description (String)
-//coverImage (String)
     private Long id;
     private String title;
     private String author;
     private String isbn;
```

Comment:
Add constraints: nullable = false, unique - true
Check other fields according to the task description

Reply:
Not fixed

Reply:
<img width="278" alt="Screenshot 2024-05-01 at 08 19 12" src="https://github.com/AGuliaiev/Spring_intro/assets/14053156/2abd5215-665b-4d02-9872-f0542f9a56b6">